templates/components/entry.html.twig

```diff
@@ -81,9 +81,9 @@
                 {% if entry.image %}
                     {% if entry.type is same as 'link' or entry.type is same as 'video' %}
                         <figure>
-                            <div class="image-filler"
-                                 style="background-image: url({{ asset(entry.image.filePath) |imagine_filter('entry_thumb') }})"
-                                 aria-hidden="true"></div>
+                            <div class="image-filler" aria-hidden="true">
+                                {{ component('blurhash_image', {blurhash: entry.image.blurhash}) }}
```

Comment:
this might need a fallback? I'm getting `Error rendering "blurhash_image" component: Expected argument of type "string", "null" given at property path "blurhash".`

Reply:
noted. opted to add a guard around blurhash filler component and provides backgroud color for `image-filler` div

it should look something like this when there's no blurhash image:

![image](https://github.com/MbinOrg/mbin/assets/20770492/efdef1cc-206b-4a4b-b5fe-3c90f039f6b7)

![image](https://github.com/MbinOrg/mbin/assets/20770492/dc5838a8-3692-414f-802f-4d3846fe1981)

Reply:
I thought it was going to be anything generated by fixtures but that seems incorrect, many fixtures have it. Looking at my db, the only thing I'm noticing is all of the heights are very tiny

<details>
<summary>db query</summary>

```
# select width, height, blurhash from image where blurhash is null;
 width | height | blurhash 
-------+--------+----------
   300 |     13 | 
   300 |     11 | 
   300 |     11 | 
   300 |      8 | 
   300 |     11 | 
   300 |      1 | 
   300 |      6 | 
   300 |      4 | 
   300 |     12 | 
   300 |     13 | 
   300 |      5 | 
   300 |      7 | 
   300 |      4 | 
   300 |      5 | 
   300 |     13 | 
   300 |      6 | 
   300 |     12 | 
   300 |      5 | 
   300 |     11 | 
   300 |     11 | 
   300 |     11 | 
   300 |      3 | 
   300 |      6 | 
   300 |      7 | 
   300 |      4 | 
   300 |     11 | 
   300 |      2 | 
   300 |      1 | 
   300 |      3 | 
   300 |      4 | 
   300 |      8 | 
   300 |      7 | 
   300 |     13 | 
   300 |     13 | 
   300 |      9 | 
   300 |      4 | 
   300 |      7 | 
   300 |     12 | 
   300 |     12 | 
   300 |      2 | 
   300 |      6 | 
   300 |      6 | 
   300 |      7 | 
   300 |      6 | 
(44 rows)

```

</details>